pantheon/src/main/java/tech/pegasys/pantheon/controller/IbftPantheonController.java

```diff
@@ -318,6 +318,11 @@ private IbftPantheonController(
           } catch (final IOException e) {
             LOG.error("Failed to close storage provider", e);
           }
+          try {
```

Comment:
We should stop the synchronizer prior to the storage.  The shutdown process could try to write more data.